### PR TITLE
delete .md in navigation page to avoid 404 in official web

### PR DESCRIPTION
**Pull request by hellolilyliuyi**

### CODE REVIEW

File: introduction/StarRocks_intro.md [hunk 1]
```diff
@@ -102,22 +102,22 @@
...
 <NavBoxPart title="参考​">
 <NavBoxPartItem>
 
-- [SQL参考](../sql-reference/sql-statements/account-management/ALTER%20USER.md)
-- [函数参考](../sql-reference/sql-functions/date-time-functions/convert_tz.md)
-- [数据类型](../sql-reference/sql-statements/data-types/TINYINT.md)
+- [SQL参考](../sql-reference/sql-statements/account-management/ALTER%20USER)
+- [函数参考](../sql-reference/sql-functions/date-time-functions/convert_tz)
+- [数据类型](../sql-reference/sql-statements/data-types/TINYINT)
 </NavBoxPartItem>
```
Comment (EsoragotoSpirit) [inline introduction/StarRocks_intro.md:121]: 上方缺少空格

File: introduction/StarRocks_intro.md [hunk 2]
```diff
@@ -59,40 +59,40 @@
...
 
-- [JSON](../sql-reference/sql-statements/data-types/JSON.md)
-- [ARRAY](../sql-reference/sql-statements/data-types/Array.md)
-- [MAP](../sql-reference/sql-statements/data-types/Map.md)
-- [STRUCT](../sql-reference/sql-statements/data-types/STRUCT.md)
+- [JSON](../sql-reference/sql-statements/data-types/JSON)
+- [ARRAY](../sql-reference/sql-statements/data-types/Array)
+- [MAP](../sql-reference/sql-statements/data-types/Map)
+- [STRUCT](../sql-reference/sql-statements/data-types/STRUCT)
 </NavBoxPartItem>
```
Comment (EsoragotoSpirit) [inline introduction/StarRocks_intro.md:75]: 上方缺少空格

Comment (hellolilyliuyi) [reply introduction/StarRocks_intro.md:75]: 为啥要有空格

**Review by EsoragotoSpirit — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
@mergify backport 3.1

**Pull request merged**